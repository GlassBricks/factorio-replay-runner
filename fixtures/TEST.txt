[Error] 151 GlassBricks imported a blueprint from the blueprint library!
[Error] 866 GlassBricks ran disallowed command: /c game.print("hi")
[Info] 6460 First rocket launched
