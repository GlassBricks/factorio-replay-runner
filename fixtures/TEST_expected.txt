[Info ]	         0	00:00:00
[Error]	       151	GlassBricks imported a blueprint from the blueprint library!
[Error]	       394	GlassBricks used map editor!
[Error]	       614	GlassBricks used map editor!
[Error]	       866	GlassBricks ran disallowed command: /c game.print("hi")
[Error]	      1122	GlassBricks used map editor!
Replay exited successfully at tick 6460: First rocket launched
